fastlane/metadata/android/fr-FR/full_description.txt

```diff
@@ -0,0 +1,21 @@
+Pour beaucoup de personnes, utiliser une ambiance légère permet de rester calme et se concentrer, parfois cela augmente aussi la productivité ou diminue le stress.
+
+Noice est une application qui permet de créer des ambiances personnalisées à partir de plusieurs sons pré-enregistrés. Créez un environnement parfait en les combinant pour vous concentrer sur votre travail ou vous relaxer avant de dormir.
+
+<b>Fonctionnalités</b>
+- 24 ambiances sonores
+- Créez et sauvegardez vos mixes personnalisés
+- Minuteur pour accompagner le sommeil
+- Peut accompagner d'autres pistes sonores
```

Comment:
For the first suggestion about sleep time I translated it a bit differently because this word does not exist in french and we just call this "minuterie". To add the "sleep" meaning I added something along the line of "waiting for sleep"

For the second suggestion about raw noise, I searched on internet how those types of noises (pink, white, brownian...) where called in french because I didn't know this word. From what I understood in french it is called "colored noise".

Reply:
Okay first one makes sense in that case. I think you've misread the second one. Please get back to me on that one.

Reply:
Yeah sorry I read the next comment and answered on another question.
For the "alongside other music players"  the "pistes" word is probably translated by google by "track" but in french while "pistes" can mean track, it can also mean any other media player.

To explain a bit my method, I took the english meaning of each sentences/expression and I tried to find an equivalent in french not translating litterally. Which is probably why google translate is sometimes finding unconsistent meaning.

I found your app not too long ago and I thought it would be great for my mother to be honest, so I am doing the translation in french for her. So my mindset was "how could I translate this in french for her to understand". In the end I think that if she understands any french person would understand.